gamedata/gamedata.json

```diff
@@ -27,7 +27,7 @@
       "deployParams": [],
       "deployFunds": 0,
       "deployId": "1",
-      "instanceGas": 350000,
+      "instanceGas": 450000,
```

Comment:
Is this file generated automatically? What are these values?

Reply:
I filled it in. Its the pre-filled gasLimit in metamask for the users to deploy a new instance. Updating to 0.6 meant some of the contracts need more gas to deploy, and therefore a higher pre-filled value.